python/lsst/ts/ofc/ofc.py

```diff
@@ -76,13 +75,13 @@ class OFC:
     PSSN: Normalized point source sensitivity.
     """
 
-    def __init__(self, ofc_data, log=None):
+    def __init__(self, ofc_data: OFCData, log: logging.Logger = None) -> None:
```

Comment:
If `log` can be of either `logging.Logger` or `None`, the type should be `logging.Logger | None`, e.g.:

```
 def __init__(self, ofc_data: OFCData, log: logging.Logger | None = None) -> None:
```